http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1127,13 +1127,15 @@ class LeoPubsubMessageSubscriber[F[_]](
       } yield ()
 
       appChart <- appQuery.getAppChart(msg.appId).transaction
+      labels <- labelQuery.getAllForResource(msg.appId.id, LabelResourceType.app).transaction
+      isAoU = labels.get(AOU_UI_LABEL).contains("true")
       _ <- asyncTasks.offer(
         Task(
           ctx.traceId,
           task,
           Some(handleKubernetesError),
           ctx.now,
-          TaskMetricsTags("createApp", Some(resolveAppType(msg.appType, appChart)), None, CloudProvider.Gcp)
+          TaskMetricsTags("createApp", Some(resolveAppType(msg.appType, appChart)), Some(isAoU), CloudProvider.Gcp)
```

Comment:
How much do we need `AoU` to be referenced specifically in the codebase? Are these AoU specific features?

Reply:
they're not AoU specific...but these tags make it possible to know whether these data points are AoU or non AoU

Reply:
not for now ... but is there another way of doing this?, besides hardcoding AoU in the codebase?

Reply:
it is something to think about for future our probably near-term future as we refactor this